src/astro/sql/operators/drop.py

```diff
@@ -17,14 +19,15 @@ def __init__(
         **kwargs,
     ):
         self.table = table
-        task_id = task_id or get_unique_task_id(table.name + "_truncate")
+        task_id = task_id or get_unique_task_id("_drop")
```

Comment:
Don't you think having `table.name` as a prefix is more meaningful to users?

Reply:
https://github.com/astronomer/astro-sdk/pull/556#issuecomment-1194869657

This was an error that made the TruncateOperator not work btw

Reply:
@dimberman would there be an alternative way of giving a more friendly default user name, without the user having to specify it?

Would we be able to use templating or something else to give users a more friendly task name in the Airflow UI?

From a user perspective, displaying "drop_table_name" is a better experience than displaying "drop_10924"